rs/tests/driver/src/util.rs

```diff
@@ -822,7 +824,9 @@ pub async fn agent_with_client_identity(
     client: reqwest::Client,
     identity: impl Identity + 'static,
 ) -> Result<Agent, AgentError> {
-    let transport = ReqwestTransport::create_with_client(url, client)?.with_use_call_v3_endpoint();
+    let transport = ReqwestTransport::create_with_client(url, client)?
+        .with_use_call_v3_endpoint()
```

Comment:
Most likely there is a race condition that needs to be fixed in a different way. Ideally we know exactly why the first request fails.

No action needed here.